<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KstDataDialog</class>
  <widget class="QDialog" name="KstDataDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>704</width>
        <height>469</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item rowspan="1" row="2" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="_ok" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="_cancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0" >
        <widget class="Q3VBox" name="_contents" >
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Plain</enum>
          </property>
        </widget>
      </item>
      <item rowspan="2" row="0" column="1" colspan="1" >
        <widget class="EditMultipleWidget" name="_editMultipleWidget" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="minimumSize" >
            <size>
              <width>300</width>
              <height>0</height>
            </size>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <layout class="QGridLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item row="0" column="1" >
            <widget class="QLineEdit" name="_tagName" />
          </item>
          <item row="0" column="0" >
            <widget class="QLabel" name="_selectLabel" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Unique name:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1" colspan="1" >
            <widget class="QLineEdit" name="_legendText" />
          </item>
          <item row="1" column="0" >
            <widget class="QLabel" name="_legendLabel" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Legend Text:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="2" >
            <widget class="QPushButton" name="_editMultiple" >
              <property name="text" >
                <string>Edit &amp;Multiple >></string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >kstdataobject.h</include>
  </includes>
  <connections>
    <connection>
      <sender>_ok</sender>
      <signal>clicked()</signal>
      <receiver>KstDataDialog</receiver>
      <slot>ok()</slot>
    </connection>
    <connection>
      <sender>_cancel</sender>
      <signal>clicked()</signal>
      <receiver>KstDataDialog</receiver>
      <slot>close()</slot>
    </connection>
  </connections>
</ui>
